bin/skonfig

```diff
@@ -1,25 +1,23 @@
 #!/usr/bin/env python3
 
-import os
-import subprocess
-import sys
-import traceback
+if __name__ == "__main__":
```

Comment:
Why? This file is always executed, not imported.

Reply:
I feel like adding `if __name__ == "__main__":` to all scripts which are meant to be executed is a good practice.

While, true, it should be obvious that you should normally not import this module, I find it beneficial because this magic line immediately tells the reader of the source code that this script is meant to be executed and what it does when it is executed.

But in this case, since everything it below the `if`, I'm not sure anymore if we should keep it after letting it sit for a few days.

Reply:
I vote for keeping the `__name__` check. The file does nothing good when imported and we're consistent with `bin/cdist`.

Reply:
I'm not against it (quite important technical detail imho :smile:) and if you strongly believe it, then let's make it happen.